src/solvers/refinement/string_constraint_generator_transformation.cpp

```diff
@@ -359,38 +360,57 @@ string_exprt string_constraint_generatort::add_axioms_for_char_set(
   return res;
 }
 
+static optionalt<std::pair<exprt, exprt>> try_char_char(
+  exprt needle, exprt replacement)
```

Comment:
`needle` and `replacement` are not very appropriate here since we don't replace anything inside this function